packages/rke2-multus/charts/templates/daemonSet.yaml

```diff
@@ -59,8 +61,8 @@ spec:
           value: {{ .Values.cniplugins.skipcnis }}
       containers:
       - name: kube-{{ .Chart.Name }}
-        image: {{ template "system_default_registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}
-        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        image: {{ template "system_default_registry" . }}{{ .Values.images.thin.repository }}:{{ .Values.images.thin.tag }}
```

Comment:
this will break users who are currently overriding the image via images.repository and images.tag. We need to continue to support the old structure (use Values.image if Values.images is nil) for backwards compatibility purposes.